pages/cart.js

```diff
@@ -0,0 +1,31 @@
+import { useQuery } from '@apollo/client'
+import { GET_CART_LINE_ITEMS_QUERY } from '../containers/line-items/queries'
+
+const CartPage = () => {
+  const { loading, error, data } = useQuery(GET_CART_LINE_ITEMS_QUERY)
+
+  if (error) {
+    return <div>Error loading menu.</div>
+  }
+
+  if (loading) {
+    return <div>Loading</div>
+  }
+
+  const { lineItems } = data.currentOrder
+
+  return (
+    <main>
+      <p>Shopping Cart</p>
+      <ul>
+        {lineItems.nodes.map(line_item => (
```

Comment:
We need an empty state for the cart (when `lineItems.nodes` is empty) like you did in `containers/line-items/index.js`.